filip/clients/ngsi_v2/cb.py

```diff
@@ -2148,9 +2111,10 @@ def does_entity_exist(self, entity_id: str, entity_type: str) -> bool:
 
     def patch_entity(
         self,
-        entity: ContextEntity,
-        old_entity: Optional[ContextEntity] = None,
-        override_attr_metadata: bool = True,
+        entity: Union[ContextEntity, ContextEntityKeyValues],
+        key_values: bool = False,
+        forcedUpdate: bool = False,
+        override_metadata: bool = False,
```

Comment:
The `patch_entity` now aligns with the official API specification https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/orion-api.md#update-existing-entity-attributes-patch-v2entitiesentityidattrs